cachito/workers/pkg_managers/npm.py

```diff
@@ -25,7 +25,7 @@
 log = logging.getLogger(__name__)
 
 
-def _get_deps(package_lock_deps, file_deps_allowlist, _name_to_deps=None):
+def _get_deps(package_lock_deps, file_deps_allowlist, _name_to_deps=None, workspaces=None):
```

Comment:
Just a boring comment: maybe it's better to have a workspace before `_name_to_deps`.

Reply:
I don't get why we have a parameter started with "_"... those are meant to imply private functions or attributes, and a parameter's scope is always the function :man_shrugging:

Reply:
Changing the order will mess with the calls to this method. `_name_to_deps` is not used as parameter with `_name_to_deps=.....` on few places so the order matters. I also don't get why is the private attribute used. Maybe I can create separate commit to address this. Wdyt?

Reply:
added second commit with changes to `name_to_deps`